synapse/federation/transport/client.py

```diff
@@ -321,6 +334,45 @@ async def send_leave_v2(self, destination, room_id, event_id, content):
 
         return response
 
+    @log_function
+    async def send_knock_v2(
```

Comment:
I think this got merged as a `v` endpoint:

```suggestion
    async def send_knock_v1(
```

Will need to update the callers too.

Reply:
Oh, good catch. Thankfully the Complement tests don't need to be updated.